src/net35/Radical/Observers/BrokerObserver.cs

```diff
@@ -77,9 +80,9 @@ public MessageBrokerMonitor( IMessageBroker broker )
         /// </summary>
         /// <typeparam name="TMessage">The type of the message.</typeparam>
         /// <returns>This monitor instance.</returns>
-        public MessageBrokerMonitor WaitingFor<TMessage>() where TMessage : class
+        public MessageBrokerMonitor WaitingFor<TMessage>( InvocationModel invocationModel = InvocationModel.Default ) where TMessage : class
```

Comment:
I can confirm that this is indeed a breaking change, so the only option is to change it to:

```
public MessageBrokerMonitor WaitingFor<TMessage>() where TMessage : class
{
   return this.WaitingFor<TMessage>( m => true, InvocationModel.Default );
}
```

removing the default parameter.